backend/apps/matching/tables/approval.py

```diff
@@ -0,0 +1,53 @@
+import django_tables2 as tables
+from django_tables2 import TemplateColumn
+
+from apps.matching.models import Participant
+from apps.matching.src.dual_factory import instanciate_for_participants
+
+approval_button = '<form>{% csrf_token %}<input type="hidden" value="{{record.uuid}}" name="uuid"> <button type="submit" name="change_approval" formmethod="post" class="btn btn-sm {% if record.is_approved %} btn-warning {% else %} btn-success {%endif%}">{% if record.is_approved %}Disapprove{% else %}  Approve {%endif%}</button></form>'
+info_button = (
+    '<a class="btn btn-info btn-sm" href="/matching/hospital_view/{{record.uuid }}">More Info </a>'
+)
+delete_button = (
+    "<!-- Button trigger modal -->"
+    '<button type="button" class="btn btn-sm btn-danger" data-toggle="modal" data-target="#exampleModal{{record.uuid}}">'
+    "Delete"
+    "</button>"
+    "<!-- Modal -->"
+    '<div class="modal fade" id="exampleModal{{record.uuid}}" tabindex="-1" role="dialog" aria-labelledby="exampleModalLabel" aria-hidden="true">'
+    '  <div class="modal-dialog" role="document">'
+    '    <div class="modal-content">'
+    '      <div class="modal-header">'
+    '        <h5 class="modal-title" id="exampleModalLabel">Are you sure?</h5>'
+    '        <button type="button" class="close" data-dismiss="modal" aria-label="Close">'
+    '          <span aria-hidden="true">&times;</span>'
+    "        </button>"
+    "      </div>"
+    '      <div class="modal-body">'
+    "        Are you sure you want to delete the user with email {{ record.user__email }}?"
```

Comment:
This does not work unfortunately:

![Screenshot from 2020-06-21 13-33-34](https://user-images.githubusercontent.com/7229585/85223623-8b48fe00-b3c4-11ea-9a56-31a81056fe8c.png)

Reply:
Fixed